src/pages/validators/index.tsx

```diff
@@ -139,26 +138,29 @@ const Validators: React.FC<PropsWithChildren> = () => {
   const [filterValidators, fetchPartialValidator, loading, setLoading] =
     useFetchPartial<IValidator>('validators', 'validator/list', 'name');
 
-  const filters: IFilter[] = [
-    {
-      title: 'Name',
-      data: filterValidators.map(validator => validator.name || ''),
-      onClick: async value => {
-        if (value === 'All') {
-          setQueryAndRouter({}, router);
-        } else {
-          setQueryAndRouter({ name: value }, router);
-        }
-      },
-      onChange: async value => {
-        setLoading(true);
-        await fetchPartialValidator(value);
+  const filters: IFilter[] = useMemo(() => {
+    return [
+      {
+        title: 'Name',
+        data: filterValidators
+          .map(validator => validator.name)
+          .filter(validator => !!validator) as string[],
```

Comment:
why not just .filter(validator => validator.name)

Reply:
![image](https://github.com/user-attachments/assets/b630aef0-8b0d-4d89-8a9f-8c5bfe8d4b4d)
![image](https://github.com/user-attachments/assets/ce558fda-cbe1-4e95-9ec0-65e1c492fdef)